From 339a573c14311259ee03f6c7235194fc07f25508 Mon Sep 17 00:00:00 2001
From: Xianzhong <xianzhong.li@nxp.com>
Date: Fri, 26 May 2017 15:50:07 +0800
Subject: [PATCH 1847/5242] MGS-2947 enable gpu subsystem for imx8qm and
 imx8qxp

commit  17e3ca18dac6bf66b62ce20fa5c48d87208487b5 from
https://source.codeaurora.org/external/imx/linux-imx.git

enable gpu subsystem device config for imx8qm and imx8qxp,
enable builtin GPU module in kernel image by default

Signed-off-by: Xianzhong <b07117@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts  |   12 ++++++++++++
 .../boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2.dts |    8 ++++++++
 2 files changed, 20 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts
index b8fe58f..22e2789 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts
@@ -486,3 +486,15 @@
 	>;
 	clocks = <&clk IMX8QM_A72_DIV>;
 };
+
+&gpu_3d0 {
+	status = "okay";
+};
+
+&gpu_3d1 {
+	status = "okay";
+};
+
+&imx8_gpu_ss {
+	status = "okay";
+};
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2.dts
index 6c75149..7ba04a4 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2.dts
@@ -253,3 +253,11 @@
 	vmmc-supply = <&reg_sd1_vmmc>;
 	status = "okay";
 };
+
+&gpu_3d0 {
+	status = "okay";
+};
+
+&imx8_gpu_ss {
+	status = "okay";
+};
-- 
1.7.9.5

